spec/views/events/show.html.erb_spec.rb

```diff
@@ -211,6 +211,24 @@
     expect(rendered).to_not have_link(t(:show_participants, scope: 'events.participants'))
   end
 
+  it "does not display send acceptances button after acceptances have been sent in selection phase" do
+    @event = assign(:event, FactoryGirl.create(:event, :in_selection_phase))
```

Comment:
> I think it is not important if the event is in selection phase.

I really disagree with that. The buttons to send acceptance and rejection emails are (atm) only displayed in selection phase, so using an event that is not in selection phase does not make sense at all, because the buttons shouldnt be displayed anyway.

>  On the other hand I think it is important that there are some accepted applications for the event. Otherwise the button could be hidden just because there are no (accepted) applications.

that could happen at some point in the future, but imo its not relevant yet, because the current implementation checks whether a mail was sent, not if there are receivers waiting for one.

In general, I dont see the point in not using the phases in tests. Not using them to specify additional values makes it only easier to get such false positives like this one would be, when the test isnt green because the code is working, but because the code to test isnt even executed